packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -53,6 +53,7 @@ export class ChannelService {
       } else {
         this.channel.appendLine(nls.localize('channel_end'));
       }
+      this.channel.appendLine('');
```

Comment:
Is this just to add an extra newline for visual aesthetics or is there a functional reason for this?

Reply:
It's just for visual aesthetic. Without this the first line of the output with the command title would not have a line separating it from the previous output.